src/utils/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'any' as a type declaration. Using 'any' defeats the purpose of using TypeScript. Instead, try to create interfaces that accurately reflect the data shapes you will be using.